modules/appeals_api/spec/support/rswag_config.rb

```diff
@@ -245,7 +245,8 @@ def schemas(api_name: nil)
     when 'appeals_status'
       merge_schemas(
         appeals_status_response_schemas,
-        generic_schemas.slice(*%i[errorModel X-VA-SSN])
+        generic_schemas.slice(*(version == 'v0' ? %i[errorModel X-VA-SSN X-VA-User] : %i[errorModel X-VA-User])),
+        shared_schemas.slice(*(version == 'v0' ? nil : %w[icn]))
```

Comment:
Had to start looked at API version here since Appeals Status v0 requires SSN and doesn't accept ICN, and v1 requires ICN and doesn't accept SSN.